shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/src/main/release-docs/NOTICE

```diff
@@ -205,3 +205,15 @@ This product bundles icons from the famfamfam.com silk icons set
 http://www.famfamfam.com/lab/icons/silk/
 Licensed under the Creative Commons Attribution Licence v2.5
 http://creativecommons.org/licenses/by/2.5/
+
+========================================================================
+
+Apache echarts NOTICE
```

Comment:
should be `Apache ECharts (incubating) NOTICE`

Reply:
ok

Reply:
line 211 should be incubating too